samples/Microsoft.Bot.Samples.EchoBot-AspNet461/Web.config

```diff
@@ -6,9 +6,9 @@
 <configuration>
   <appSettings>
     <!-- update these with your BotId, Microsoft App Id and your Microsoft App Password-->
-    <add key="BotId" value="YourBotId" />
-    <add key="MicrosoftAppId" value="" />
-    <add key="MicrosoftAppPassword" value="" />
+    <add key="BotFramework.BotId" value="YourBotId" />

```

Comment:
Is this same change not needed in the AlarmBot json files?

Reply:
Actually, I looked into what this setting was about and I can't find anything using it. I actually intended to remove it altogether. Are you aware of anything actually using it that I'm missing???

Reply:
**RE: AlarmBot** - no, because those are NetCore and they're using the `ConfigurationCredentialProvider` which I didn't touch (for now), so it still tries to load the "old" names from configuration.